SecurityUtil.doAsPrivilege=An exception occurs when running the PrivilegedExceptionAction block.

